### ISSUE TITLE
Getting all map layers

### ISSUE THREAD

**Issue by deepal13** [BrianBunker/cmv-widgets#37]
@tmcgee I am trying to get all the layers using map.layerIds.length and then iterating through them. It gives me all the layers except from the ones added through DnD widget. How can I get those layers?

Also, The layers added through DnD doesn't move up or down with respect to already added operational layer. Can you please help me?

Thanks

**Comment by tmcgee** [comment]
@deepal13 These are graphic layers so you have to use [`map.graphicsLayerIds`](https://developers.arcgis.com/javascript/3/jsapi/map-amd.html#graphicslayerids)

**Comment by deepal13** [comment]
@tmcgee That worked fine. Thanks :)
The layers added through DnD doesn't move up or down with respect to already added operational layer. Is it supposed to be this way?

**Comment by tmcgee** [comment]
@deepal13 In the Esri JavaScript API, graphic layers are always drawn on top so they cannot be moved below other non-graphic layers.

**Comment by deepal13** [comment]
@tmcgee Ok. But I am unable to move San Francisco 311 Incidents below louisvillePubSafety in the demo example.

**Comment by tmcgee** [comment]
@deepal13 yes, that is how it works. The San Francisco 311 layer is a feature layer (graphic) and the Louisville Public Safety layer is a dynamic layer (non-graphic)

**Comment by deepal13** [comment]
@tmcgee Ok. I got it. Thanks for your help.
